opentelemetry-appender-tracing/benches/logs.rs

```diff
@@ -173,5 +174,16 @@ fn criterion_benchmark(c: &mut Criterion) {
     benchmark_with_noop_layer(c, false, "noop_layer_disabled");
 }
 
-criterion_group!(benches, criterion_benchmark);
+#[cfg(not(target_os = "windows"))]
+criterion_group! {
+    name = benches;
+    config = Criterion::default().with_profiler(PProfProfiler::new(100, Output::Flamegraph(None)));
+    targets = criterion_benchmark
+}
+#[cfg(target_os = "windows")]
+criterion_group! {
+    name = benches;
+    config = Criterion::default();
+    targets = criterion_benchmark
+}
```

Comment:
May be we can use https://crates.io/crates/cfg-if to make things easier. Especially in dev tools